src/test/java/com/mariamkatamashvili/trainerwork/messaging/MessageConsumerTest.java

```diff
@@ -0,0 +1,45 @@
+package com.mariamkatamashvili.trainerwork.messaging;
+import com.mariamkatamashvili.trainerwork.config.TestConfig;
```

Comment:
It would be good to store steps/definitions classes procesors under correspondent package.